event_registration_multi_qty/README.rst

```diff
@@ -0,0 +1,98 @@
+============================
+Event Registration Multi Qty
+============================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fevent-lightgray.png?logo=github
+    :target: https://github.com/OCA/event/tree/12.0/event_registration_multi_qty
+    :alt: OCA/event
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/event-12-0/event-12-0-event_registration_multi_qty
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/199/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module allows to make registration to events with more than one attendee.
+
+This was the unique behavior on versions 8 and 9. As it's useful for some use
+cases, this module allows both ways of registering attendees.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+To configure the possibility of registering mutiple attendees:
+
+#. Go to *Events > Events*.
+#. Create one event.
+#. Go to *Registration tab* and check "Registration Multi Qty" to activate
```

Comment:
LGTM but seems this need update. I tested on Runbot and the option is now the default behavior. 

![image](https://user-images.githubusercontent.com/16461806/55245388-52fb5400-5243-11e9-8da6-344cd2bb705a.png)